lib/semantics/semantics.h

```diff
@@ -135,6 +136,18 @@ class SemanticsContext {
   parser::Message &Say(parser::Message &&msg) {
     return messages_.Say(std::move(msg));
   }
+  template<typename... A>
```

Comment:
I just basically wrote the same thing for use in `lib/evaluate`.

Reply:
Hmmm.  Where's the best place for this function to live?

Reply:
The library dependence order is common -> parser -> evaluate -> semantics, so probably evaluate.  But leave it here for now, I'm unlikely to merge before you do, and I can clean it up later.

Reply:
It looks like at least this function could call `evaluate::AttachDeclaration` to attach the appropriate message.

Reply:
Thanks, Tim.  I'll use ```evaluate::AttachDeclaration```.